### DISCUSSION TITLE
Error when downloading video

### DISCUSSION THREAD

**Discussion by lordforce** [aclap-dev/video-downloadhelper#757] [Help and Q&A]
### Your question

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://avple.tv/video/100001138381",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [tcp @ 000001953cddb0c0] Failed to resolve hostname videofedbf.oudthn.com: No address associated with hostname\r\n[in#0 @ 000001953e65b240] Error opening input: I/O error\r\nError opening input file https://videofedbf.oudthn.com/decry/vd/20240422/YTg2OTU0NmRiN/005913/720_1270/libx/hls/encrypt/index.m3u8.\r\nError opening input files: I/O error\r\n",
    "id": "coapp_failure_5ab1322a-d816-4b39-a58e-51c16f120105",
    "downloadable_id": "downloadable_358309802497690"
  }
}

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://avple.tv/video/100001138381",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [tcp @ 000001953cddb0c0] Failed to resolve hostname videofedbf.oudthn.com: No address associated with hostname\r\n[in#0 @ 000001953e65b240] Error opening input: I/O error\r\nError opening input file https://videofedbf.oudthn.com/decry/vd/20240422/YTg2OTU0NmRiN/005913/720_1270/libx/hls/encrypt/index.m3u8.\r\nError opening input files: I/O error\r\n",
    "id": "coapp_failure_5ab1322a-d816-4b39-a58e-51c16f120105",
    "downloadable_id": "downloadable_358309802497690"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0.1 64-bit, licensed VDH 9.0.2.3a1 beta, CoApp 2.0.19.

I visited that page & VDH presented me with 2 MP4s, both of duration 1:06:12, both RawHLS.  I didn't know what the difference was so I downloaded them both.  The web site was throttling my downloads severely.  I was getting a few hundred thousand bytes per second with each one.  That is horrifically slow.  I let the downloads run & busied myself with other things.  They ran for about a half hour each, completing almost simultaneously.  I ended up with 2 identical MP4s, size 603M, resolution 720x1270, duration 1:06:11.  Even the bit rates were the same.  This is a porno so I'm not showing any images.  I played both videos in VLC.  I didn't sit & watch them, just sampled them at intervals to make sure they were fine.  There was audio but I had to crank the volume up to a ridiculously high level to hear anything at all.  It was a bit muffled, plus I assume it was in a language I don't speak.  I turned up the volume only long enough to hear something, then turned it back down since I wasn't actually interested in this video.

The error you got makes me speculate that your antivirus, your firewall, or both are interfering with the normal operation of VDH.  Search this forum for more discussion of that topic.  You need to follow the advice you'll find.  Also, it seems VDH does not work today though a proxy.  If you're using a proxy, you should try to go around the proxy to do VDH downloads.  I think Paul is working on that issue, or at least, it's on his todo list.
